masterlist.yaml

```
@@ -2302,6 +2302,30 @@ plugins:
         udr: 4
 
 ###### Gameplay - Combat ######
+  - name: 'Forever Radiation Lite.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/48231/' ]
```

A regex can be used to apply the location to all three ESPs.

```suggestion
  - name: 'Forever Radiation (Lite|Hardcore|Even HarderCore)\.esp'
    url: [ 'https://www.nexusmods.com/fallout4/mods/48231/' ]
  - name: 'Forever Radiation Lite.esp'
```